### PR TITLE
Add fx support

### PR DESCRIPTION
**Pull request by alter-eggo**
# Description
This pr adds FX pool type support on polygon and branded redirect card for xave pool

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?
- go to https://feat-fx-support.balancer-d6h.pages.dev/#/polygon/pool/0x726e324c29a1e49309672b244bdc4ff62a270407000200000000000000000702
- there should be branded redirect card for xave

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/components/contextual/pages/pool/PoolChart.vue [hunk 1]
```diff
@@ -253,6 +253,8 @@ function getFeesData(
       } else {
         prevValue = parseFloat(periodSnapshots[idx + 1].swapFees);
       }
+      console.log(value, prevValue);
```
Comment (garethfuller) [inline src/components/contextual/pages/pool/PoolChart.vue:256]: Remove?

File: src/components/pool/branded-redirect/constants.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+type BrandedRedirects = Record<string, BrandedRedirectData>;
+
+const POLYGON_BRANDED_REDIRECTS: BrandedRedirects = {
+  '0x726e324c29a1e49309672b244bdc4ff62a270407000200000000000000000702': {
+    id: 'xave',
+    title: 'brandedRedirect.xave.title',
+    description: 'brandedRedirect.xave.description',
+    btnText: 'brandedRedirect.xave.btnText',
+    link: 'https://xave.co/',
+  },
```
Comment (garethfuller) [inline src/components/pool/branded-redirect/constants.ts:21]: It's likely that we will end up with a lot of cases where there is many pools under a single brand. So ideally we map the `BrandedRedirectData` to a key, something like:
```suggestion
  'xave': {
    id: 'xave',
    title: 'brandedRedirect.xave.title',
    description: 'brandedRedirect.xave.description',
    btnText: 'brandedRedirect.xave.btnText',
    link: 'https://xave.co/',
  },
```
And then elsewhere we map all the relevant pool IDs to that key, e.g.
```
'xave': ['0x726e324c29a1e49309672b244bdc4ff62a270407000200000000000000000702']
```

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: src/components/_global/BalImage/BalImage.vue [hunk 3]
```diff
@@ -45,6 +45,7 @@ function onLoaded() {
         v-show="loaded"
         :width="props.width"
         :height="props.height"
+        style="max-width: 100%; max-height: 100%"
```
[the remark this answers is not in the document]
Comment (alter-eggo) [reply src/components/_global/BalImage/BalImage.vue]: removed it, no need in this here

File: src/components/pool/branded-redirect/constants.ts (hunk 2 above)
Comment (alter-eggo) [reply src/components/pool/branded-redirect/constants.ts]: this file now contains only brand data. 
pools that should have branded redirect are only here `src/constants/pools.ts` in `BrandedRedirect` field

**Review by pkattera — APPROVED** [review]
Looks good to me from a visual standpoint.

**Review by garethfuller — APPROVED** [review]

**Pull request merged**